.circleci/config.yml

```diff
@@ -492,6 +492,7 @@ workflows:
           requires:
             - ktlint_check
             - detekt
+      - firebase_screenshots
```

Comment:
Is it necessary since we're transitioning to GH actions anyway?

Reply:
It's just for generating now, that I can send the screenshots for review. It will be removed afterwards, thats why I added the "do not merge" tag.

Reply:
Please move this ?